config/conneg.yml

```diff
@@ -3,28 +3,36 @@ compression: true
 ns_definitions:
   - prefix: "/ns/core"
     core_version: "3.0"
-    html_uri: "https://docs.oasis-open-projects.org/oslc-op/core/v3.0/ps01/core-vocab.html"
+    html_uri: "https://docs.oasis-open-projects.org/oslc-op/core/v3.0/os/core-vocab.html"
     files:
       rdfxml: "core/core-vocab.rdf"
       ntriples: "core/core-vocab.nt"
       jsonld: "core/core-vocab.jsonld"
       turtle: "core/core-vocab.ttl"
   - prefix: "/ns/core/shapes/3.0"
     core_version: "3.0"
-    html_uri: "https://docs.oasis-open-projects.org/oslc-op/core/v3.0/ps01/core-shapes.html"
+    html_uri: "https://docs.oasis-open-projects.org/oslc-op/core/v3.0/os/core-shapes.html"
     files:
       rdfxml: "core/core-shapes.rdf"
       ntriples: "core/core-shapes.nt"
       jsonld: "core/core-shapes.jsonld"
       turtle: "core/core-shapes.ttl"
   - prefix: "/ns/config"
-    core_version: "2.0" # these are old files
-    html_uri: "https://oslc-op.github.io/oslc-specs/specs/config/config-vocab.html"
+    core_version: "1.0"
```

Comment:
I think this should be 3.0. We use it to send the right `OSLC-Core-Version` header.